digital_land/phase/priority.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from .phase import Phase
+from digital_land.configuration.main import Config
+
+
+class PriorityPhase(Phase):
+    """
+    Deduce priority of the entry when assembling facts
+    """
+
+    def __init__(self, config: Config = None):
+        # if not connection:s
+        # TODO causes errors when not supplied i think we should add this default in
+        # a higher up level
+        # connection = sqlite3.connect("var/cache/pipeline.sqlite3")
+        self.default_priority = 2
```

Comment:
what's the explanation for choosing 2 as this default priority? could maybe have a comment describing it.

and do we want to be able to pass in a different value for default priority?

Reply:
Currently the way it's set up we only set it to 1 if the organisation matches. In future we may want to change this default to a lower number if we want to adjust for more levels of priority.

I've added it in purely because there are multiple paces where default priority is needed so it stops. the need. to. change it multiple times